### PR TITLE
use gravatars as a fallback

### PR DESCRIPTION
**Pull request by nihil2501**

### CODE REVIEW

File: app/helpers/member_helper.rb [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-    elsif member.github_name.present?
-      url = "http://github.com/#{member.github_name}"
-    end
+    url = member.twitter_url || member.github_url
 
-    image = image_tag "members/#{member.avatar_file}", alt: member.name, title: title
+    if member.avatar_file_path.present?
+      image = image_tag member.avatar_file_path, alt: member.name, title: member.name
+    else
+      image = gravatar_image_tag member.email, alt: member.name, title: member.name, gravatar: { size: 200 }
```
Comment (jjulian) [inline app/helpers/member_helper.rb]: Where does 200 come from?

File: app/helpers/member_helper.rb [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
   def member_avatar(member)
     return nil if member.nil?
 
-    title = "#{member.name}"
-    if member.twitter.present?
-      url = "http://twitter.com/#{member.twitter}"
-    elsif member.github_name.present?
-      url = "http://github.com/#{member.github_name}"
-    end
+    url = member.twitter_url || member.github_url
```
Comment (jjulian) [inline app/helpers/member_helper.rb]: +1 I like this

File: app/helpers/member_helper.rb (hunk 1 above)
Comment (nihil2501) [inline app/helpers/member_helper.rb]: only mentioned in the readme. i could make it a constant for style points

**Comment by jjulian** [comment]
LGTM, and not just because Oren is sitting right next to me

File: app/helpers/member_helper.rb (hunk 1 above)
Comment (jjulian) [inline app/helpers/member_helper.rb]: OK, gotcha. Avatars are all 200px

File: app/helpers/member_helper.rb (hunk 2 above)
Comment (solipet) [inline app/helpers/member_helper.rb]: +1 esp since he got rid of the unused "title" var

**Comment by nihil2501** [comment]
@jjulian can this be merged?

File: app/models/member.rb [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
-      Hashie::Mash.new(:name =>        a['name'],
-                       :github_name => a['github_name'],
-                       :twitter =>     a['twitter'],
-                       :avatar_file => a['avatar_file'])
-    }
+    MEMBER_CONFIG.map(&method(:new))
+  end
+
+  def avatar_file_path
+    "member/#{avatar_file}" if avatar_file.present?
```
Comment (nerdEd) [inline app/models/member.rb]: typo here that breaks all the existing file based avatars should be `"members/#{avatar_file}" if avatar_file.present?` **members**

Comment (nerdEd) [inline app/models/member.rb]: @nihil2501 ^^

Comment (nihil2501) [inline app/models/member.rb]: @nerdEd fixed!

**Comment by nihil2501** [comment]
@nerdEd fixed

**Comment by nihil2501** [comment]
@nerdEd merge this plz!

**Comment by nerdEd** [comment]
@nihil2501 something goofy with the styles and it breaks the picture grid when the avatar comes from gravatar

**Comment by nihil2501** [comment]
good find. fixed

**Pull request merged**